pkg/csi_driver/multishare_ops_manager.go

```diff
@@ -690,15 +690,14 @@ func (m *MultishareOpsManager) listMatchedInstances(ctx context.Context, req *cs
 // 7, Both source and target instance should have the same connect mode.
 // 8. Both source and target instance should have the same KmsKeyName.
 func isMatchedInstance(source, target *file.MultishareInstance, req *csi.CreateVolumeRequest) (bool, error) {
-	if scPrefixSource, ok := source.Labels[util.ParamMultishareInstanceScLabelKey]; ok {
-		if _, ok := target.Labels[util.ParamMultishareInstanceScLabelKey]; !ok {
-			return false, fmt.Errorf("label %q missing in target instance %+v", util.ParamMultishareInstanceScLabelKey, target)
+	matchLabels := [3]string{util.ParamMultishareInstanceScLabelKey, tagKeyClusterLocation, tagKeyClusterName}
+	for _, labelKey := range matchLabels {
+		if _, ok := target.Labels[labelKey]; !ok {
```

Comment:
what about source missing the key? I think we should handle that too

Reply:
yeah that's exactly what's done here. See L698

Reply:
Ack